sections/shared/modals/SelectCurrencyModal.tsx

```diff
@@ -219,45 +221,6 @@ export const SelectCurrencyModal: FC<SelectCurrencyModalProps> = ({
 					}
 					scrollableTarget="scrollableDiv"
 				>
-					<RowsHeader>
-						<span>
-							{assetSearch ? (
-								<span>{t('modals.select-currency.header.search-results')}</span>
-							) : synthCategory != null ? (
-								t('modals.select-currency.header.category-synths', {
-									category: synthCategory,
-								})
-							) : (
-								t('modals.select-currency.header.all-synths')
-							)}
-						</span>
-						<span>{t('modals.select-currency.header.holdings')}</span>
-					</RowsHeader>
-					{synthBalancesLoading ? (
-						<Loader />
-					) : synthsResults.length > 0 ? (
-						// TODO: use `Synth` type from contracts-interface
-						synthsResults.map((synth) => {
-							const currencyKey = synth.name;
-							return (
-								<CurrencyRow
-									key={currencyKey}
-									onClick={() => {
-										onSelect(currencyKey, false);
-										onDismiss();
-									}}
-									balance={synthBalancesMap[currencyKey as CurrencyKey]}
-									token={{
-										name: synth.description,
-										symbol: synth.name,
-										isSynth: true,
-									}}
-								/>
-							);
-						})
-					) : (
-						<EmptyDisplay>{t('modals.select-currency.search.empty-results')}</EmptyDisplay>
-					)}
 					{oneInchTokensPaged.length ? (
```

Comment:
The infinite loading issue hasn't been fixed. Please check if the length of `oneInchTokensPaged` is different from `oneInchTokenList`.
![image](https://user-images.githubusercontent.com/4819006/217061493-44f54e1a-8fb8-46ed-87bf-040afa7dc1e2.png)

Reply:
This has been resolved after re-deployment.